src/main/java/org/jcodec/containers/dpx/DPXMetadata.java

```diff
@@ -0,0 +1,26 @@
+package org.jcodec.containers.dpx;
+
+import static java.lang.String.format;
+
+public class DPXMetadata {
+    public FileInfoHeader fileInfo;
+    public ImageInformationHeader imageInfo;
+    public ImageSourceInfoHeader imageSource;
+    public FilmInformationHeader filmInfo;
+    public TelevisionInfoHeader tvInfo;
+    public String userId;
+
+    public static String smpte_tc(int tcsmpte, boolean prevent_dropframe) {
```

Comment:
please do not use `under_score_naming` use java standard `camelCaseNaming` especially so in methods declared `public`